src/osd/osd_types.cc

```diff
@@ -3714,10 +3719,7 @@ void pg_notify_t::dump(Formatter *f) const
 
 void pg_notify_t::generate_test_instances(list<pg_notify_t*>& o)
 {
-  o.push_back(new pg_notify_t(shard_id_t(3), shard_id_t::NO_SHARD, 1, 1,
-			      pg_info_t(), PastIntervals()));
-  o.push_back(new pg_notify_t(shard_id_t(0), shard_id_t(0), 3, 10,
```

Comment:
What's the reason behind stripping these test instances?

Reply:
i missed that change, hit some segfault and didn't place it back, added it back